app/views/admin/promotional_features/index.html.erb

```diff
@@ -1,39 +1,54 @@
-<% page_title @organisation.name %>
-
-<div class="organisation-header">
-  <h1><%= @organisation.name %></h1>
-  <%= view_on_website_link_for @organisation %>
-</div>
-
-<section class="organisation-details">
-  <%= tab_navigation_for(@organisation) do %>
-
-    <p class="warning">Warning: changes to promotional features appear instantly on the live site.</p>
-
-    <% if @promotional_features.many? %>
-      <p><%= link_to "Reorder promotional features", reorder_admin_organisation_promotional_features_path(@organisation) %></p>
-    <% end %>
-
-    <% if @promotional_features.present? %>
-      <table class="table table-bordered">
-        <thead>
-          <tr class="table-header">
-            <th>Promotional Feature</th>
-            <th>Actions</th>
-          </tr>
-        </thead>
-
-        <tbody>
-          <%= render @promotional_features, organisation: @organisation %>
-        </tbody>
-      </table>
-    <% else %>
-      <p class="no-content no-content-bordered">No promotional features</p>
-    <% end %>
-
-    <div class="form-actions">
-      <%= link_to "New promotional feature", new_admin_organisation_promotional_feature_path, class: "btn btn-lg btn-primary"%>
-    </div>
-
-  <% end %>
-</section>
+<% content_for :page_title, @organisation.name %>
+<% content_for :context, "Organisation" %>
+<% content_for :title, @organisation.name %>
+<% content_for :title_margin_bottom, 4 %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+<p class="govuk-body">
+  <%= view_on_website_link_for @organisation, class: "govuk-link" %></p>
+
+<%= render "components/secondary_navigation", {
+  aria_label: "Promotional features navigation tabs",
+  items: secondary_navigation_tabs_items( @organisation, request.path)
+} %>
+
+<%= render "govuk_publishing_components/components/warning_text", {
+  text: "Changes to promotional features appear instantly on the live site."
+} %>
+
+<%= render "govuk_publishing_components/components/button", {
+  text: "New promotional feature",
+  href: new_admin_organisation_promotional_feature_path,
+  margin_bottom: 6,
+} %>
+
+<% if @promotional_features.many? %>
+  <p class="govuk-body">
+    <%= link_to "Reorder promotional features", reorder_admin_organisation_promotional_features_path(@organisation), class: "govuk-link", target: "_blank" %>
```

Comment:
I don't think this needs the `target: "_blank" `. Generally we don't open links on a new tab unless there's a good reason to as it's hard for screen reader users to follow what's going on.

When we do we would indicate that a new tab is being opened by doing something like

`"Reorder promotional features (opens in new tab)"`

For this though.i think it's totally fine it does it in the same tab